src/boot/prom.mjs

```
@@ -38,12 +38,20 @@ import {
 
 
 async function promBootStageZero(dispatch /*, getState */) {
-  const savedBlockchainRedux = (await promLoadReduxBlockchain()).result;
+  let savedBlockchainRedux = { };
```

So this just kind of hides the error, right?
It shouldn't fail, so at least a `console.log`, so we know to fix it.
Ideally, I fix the root cause.

---

no it was meant to try to load any data if any was saved and can be loaded.
it doesn't hide the error. that's why i set it to empty object before. the result is guaranteed to be an object and it may have data depending if data was validly saved or not.

---

(we would need to call saveReduxBlockchain somewhere for a subsequent load to work and we have not done that yet afaik)

---

I went to check this in the wallet and see yes the error is NotFound...
Some part of me thinks we should handle that better in `promLoadReduxBlockchain` to return `result: {}` instead of error if it is empty, and throw an error if eg. the system doesn't support level-js.

But there may be other side effects there... I will take your approach as actually handling NotFound and not just masking the error. Sorry for previous comment.